kubectl-plugin/pkg/cmd/log/log.go

```diff
@@ -52,6 +52,12 @@ var (
 
 		# Download logs from a RayCluster, but only for the head node
 		kubectl ray log my-raycluster --node-type head
+
+		# Download logs from a RayCluster, but only for the worker nodes
+		kubectl ray log my-raycluster --node-type worker
+
+		# Download all (worker node and head node) the logs from a RayCluster
+		kubectl ray log my-raycluster --node-type all
```

Comment:
I don't know why I cannot comment on unchanged lines, but I think on lines 47 to 48 we should write down the default behavior if `--node-type` is not provided by user.

Reply:
Done!